server/apps/memory-app/README.md

```diff
@@ -32,6 +32,14 @@ keytool -genkey -alias james -keyalg RSA -keystore keystore
 docker run -v $PWD/keystore:/root/conf/keystore docker run apache/james:cassandra-latest
 ```
 
+In the case of quick start James without manually creating a keystore (e.g. for development), just input the command argument `start-dev` when running,
+James will auto-generate keystore file with the default setting:
+
+```
+docker run apache/james:memory-latest start-dev
```

Comment:
```suggestion
docker run apache/james:memory-latest --generate-keystore
```

Is more idiomatic from command line, and likely more self explanatory.

Reply:
We should document first how to auto-generate the keystore, then explain how to set up your own...

Reply:
Please document `--generate-keystore` in all other applications.